packages/amplify-appsync-simulator/src/__tests__/utils/graphql-runner/query-and-mutation.test.ts

```diff
@@ -144,7 +144,7 @@ describe('runQueryAndMutation', () => {
       }
     `);
 
-    const error = new GraphQLError('An error from template');
+    const error = new Error('An error from template');
```

Comment:
Why is this replacement necessary?

Reply:
Was a placeholder and I forgot to commit the real fix. I'll push it shortly.